src/main/java/com/doongji/nestalk/email/EmailService.java

```diff
@@ -0,0 +1,33 @@
+package com.doongji.nestalk.email;
+
+import com.doongji.nestalk.entity.user.User;
+import com.doongji.nestalk.error.NotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+
+@RequiredArgsConstructor
+@Service
+public class EmailService {
+
+    private final JavaMailSender mailSender;
+
+    public void mailSend (String to, String sub, String text){
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setTo(to);
+        message.setSubject(sub);
+        message.setText(text);
+        mailSender.send(message);
+    }
+
+    public String sendTemporaryPassword(User user) throws NotFoundException{
+        UUID uuid = UUID.randomUUID();
+        String temp = uuid.toString().substring(0, 8);
```

Comment:
```suggestion
       String temp = UUID.randomUUID().toString().substring(0, 8);
```
이렇게 한줄로 줄이면 깔끔할것같아요!

Reply:
효진님이 좋은 부분 이야기해주신 것 같아요. 개인적으로 직감적으로 와닿는다는 생각이 들었습니다. 이렇게 사용할 수 있구나 하나 배워갑니다